lib/models/user-whitelist.js

```diff
@@ -13,4 +14,52 @@ util.inherits(UserWhitelist, Base);
 
 require('../extend-with-factories')(UserWhitelist);
 
+/**
+ * Is an organization in a trial period?
+ * @returns {boolean}
+ */
+UserWhitelist.prototype.isInTrial = function () {
+  return moment(this.attrs.trialEnd) > moment.utc();
+};
+
+/**
+ * Is the organization in a grace period?
+ * @returns {boolean}
+ */
+UserWhitelist.prototype.isInGrace = function () {
+  return !this.isInTrial() && moment(this.attrs.gracePeriodEnd) > moment.utc();
+};
+
+/**
+ * Is the organization in an active period?
+ * @returns {boolean}
+ */
+UserWhitelist.prototype.isInActivePeriod = function () {
+  return moment(this.attrs.activePeriodEnd) > moment.utc();
+};
+
+/**
+ * Has the organizations grace period expired?
+ * @returns {boolean}
+ */
+UserWhitelist.prototype.isGraceExpired = function () {
+  return !this.isInTrial() && moment(this.attrs.gracePeriodEnd) < moment.utc();
+};
+
+/**
+ * How many days are remaining in the trial?
+ * @returns {Integer} - Days remaining in trial
+ */
+UserWhitelist.prototype.trialDaysRemaining = function () {
+  return moment(this.attrs.trialEnd).diff(moment.utc(), 'days');
```

Comment:
I don't think you need the `utc` in these. You can just do `moment()` since the "time" (whatever that means) is the same.